integration_tests/pubspec.yaml

```diff
@@ -1,20 +1,6 @@
-# Copyright 2022 Xayn AG
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
```

Comment:
As far as I can tell this is the standard way to indicate license information in dart, or at least I have seen this in multiple projects.

Reply:
I checked, blue is not doing that and I look at the most popular packages and top dart packages on pub.dev and no one is doing this. Seems that pub.dev read the licence from the LICENCE file in the repo